manifests/init.pp

```diff
@@ -93,6 +99,7 @@
   String[1] $python_package_name = 'python3-clustershell',
   Stdlib::Absolutepath $conf_dir = '/etc/clustershell',
   Stdlib::Absolutepath $conf = '/etc/clustershell/clush.conf',
+  Stdlib::Absolutepath $clush_conf_dir = '/etc/clustershell/clush.conf.d',
```

Comment:
Missing doc string.

Reply:
Added in 4dbb8f5658ed066273ff781a91a6469be12a7ebb